reports/sks444.md

```diff
@@ -0,0 +1,253 @@
+    - details: >
+      Add teams to Contributor Model
+  - 9:
+    - repo: IGitt
+    - link: https://gitlab.com/gitmate/open-source/IGitt/commit/b3caeceb
+    - details: >
+      Repository: Add filter_merge_requests method
+  - 10:
+    - repo: webservices
+    - link: https://gitlab.com/coala/landing/commit/bafb226b
+    - details: >
+      backend/db.sqlite3: Add old database files
+  - 11:
+    - repo: webservices
+    - link: https://gitlab.com/coala/landing/commit/29c2fc4c
+    - details: >
+      org/: Only pass valid usernames to methods
+  - 12:
+    - repo: webservices
+    - link: https://gitlab.com/coala/landing/commit/85372c2d
+    - details: >
+      org/cron.py: Fix contributors cron job
+  - 13:
+    - repo: webservices
+    - link: https://gitlab.com/coala/landing/commit/d68a6684
+    - details: >
+      Serialize coala mrs on GitHub and GitLab
+  - 14:
+    - repo: community
+    - link: https://github.com/coala/community/commit/d8e760bf
+    - details: >
+      data/tests/: Add tests for data app
+  - 15:
+    - repo: community
+    - link: https://github.com/coala/community/commit/baae2218
+    - details: >
+      test_management_commands.py: Fix webservices error
+  - 16:
+    - repo: community
+    - link: https://github.com/coala/community/commit/e09ae105
+    - details: >
+      data/: Fix import_contributors_data command
+  - 17:
+    - repo: community
+    - link: https://github.com/coala/community/commit/829ceba7
+    - details: >
+      test_models.py: Fix tests
+  - 18:
+    - repo: community
+    - link: https://github.com/coala/community/commit/34748e4b
+    - detials: >
+      Get 100% test coverage for the data app
+  - 19:
+    - repo: webservices.coala.io
+    - link: https://github.com/sks444/webservices.coala.io/commit/9d263725
+    - details: >
+      Generate a fake api similar to coala's webservices
+  - 20:
+    - repo: community
+    - link: https://github.com/coala/community/commit/537602ec
+    - details: >
+      data/: Import IGitt data
+  - 21:
+    - repo: community
+    - link: https://github.com/coala/community/commit/68556ccb
+    - details: >
+      data/: Add url field to Issue and MergeRequest model
+  - 22:
+    - repo: community
+    - link: https://github.com/coala/community/commit/ef814f6e
+    - details: >
+      Add gamification app
+  - 23:
+    - repo: community
+    - link: https://github.com/coala/community/pull/153/commits/7bca77cb
+    - details: >
+      [WIP] openhub/tests/: Create tests for openhub app
+  - 24:
+    - repo: community
+    - link: https://github.com/coala/community/pull/145/commits/b1ca875c
+    - details: >
+      [WIP] gitter/: Messages, Analysis and Import
+  - 25:
+    - repo: community
+    - link: https://github.com/coala/community/pull/152/commits/3c717312
+    - details: >
+      [WIP] openhub/data.py: Convert data dict to list
+  - 26:
+    - repo: igitt-django
+    - link: https://gitlab.com/sks444/igitt-django/commit/824ced17
```

Comment:
reviewed https://gitlab.com/sks444/igitt-django/commit/824ced17

looks like a few issues and one or two MRs are needed.

Reply:
updated

Reply:
not nearly done.  needs to support pgsql and sqlite.  tests need to show this.